modules/core/plan/src/Form/PlanTypeForm.php

```diff
@@ -16,6 +16,13 @@
  */
 class PlanTypeForm extends EntityForm {
 
+  /**
+   * The plan type entity.
+   *
+   * @var \Drupal\plan\Entity\PlanTypeInterface
+   */
+  protected $entity;
+
```

Comment:
I have always noticed that it was common for core/contrib to override this `$entity` parameter in forms extending `EntityForm`. This is an easy way to provide additional type checking for our custom entity types across all usages in this form file.